README.md

```diff
@@ -210,13 +244,13 @@ Creates a query to update one entity.
 
 - table: the table name
 - updatableFields: the fields that can be updated
-- idFields: the fields used to select the target entity (default: `id`)
+- idField: the field to use to select each document
+- idFields: array of idField, to use in place of idField if table has several ids.
```

Comment:
wouldn't it be better to only have `idFields` and document that we can pass only one ?